modules/apps/object/object-test/src/testFunctional/macros/CreateObject.macro

```diff
@@ -226,4 +226,17 @@ definition {
 			locator1 = "CreateObject#VIEW_OBJECT");
 	}
 
+
+	macro chooseEntryRelationship {
```

Comment:
Hi @vsigor ,

You can change the macro name to `chooseEntryOnRelationshipField`

Thanks.